radixdlt-core/radixdlt/src/main/java/com/radixdlt/client/ClientApiModule.java

```diff
@@ -47,7 +47,7 @@ public void configure() {
 	}
 
 	@ProvidesIntoMap
-	@StringMapKey("radix.universeMagic")
+	@StringMapKey("radix.networkId")
 	public JsonRpcHandler universeMagicHandler(HighLevelApiHandler highLevelApiHandler) {
```

Comment:
maybe rename to `networkIdHandler`

Reply:
there are other things to clean up, this one will be fixed along with them.